### PR TITLE
makefile,store/tikv: allow golint check context.Context in 'make check'

### PR DESCRIPTION
**Pull request by tiancaiamao**
Since `context.Context` is the first parameter now, we can turn on the golint check in `make check`

@coocood @zhexuany

### CODE REVIEW

**Comment by coocood** [comment]
How about renaming `goctx` back to 'context'?

**Comment by coocood** [comment]
LGTM

File: Makefile [hunk 1]
```diff
@@ -78,7 +78,7 @@ check: errcheck
 	@ go tool vet -all -shadow $(TOPDIRS) 2>&1 | awk '{print} END{if(NR>0) {exit 1}}'
 	@ go tool vet -all -shadow *.go 2>&1 | awk '{print} END{if(NR>0) {exit 1}}'
 	@echo "golint"
-	@ golint ./... 2>&1 | grep -vE 'vendor|context\.Context|LastInsertId|NewLexer|\.pb\.go' | awk '{print} END{if(NR>0) {exit 1}}'
+	@ golint ./... 2>&1 | grep -vE 'vendor' | awk '{print} END{if(NR>0) {exit 1}}'
```
Comment (jackysp) [inline Makefile]: I think we could use `golint -set_exit_status $(PACKAGES)`.

**Comment by jackysp** [comment]
LGTM

**Pull request merged**